docs/source/examples/demo_tracing_model_torchscript_onnx.ipynb

```diff
@@ -0,0 +1,463 @@
+    "\n",
+    "\n",
+    "Users need to provide a model id from sentence transformers (an example: `sentence-transformers/all-MiniLM-L6-v2`). `save_as_onnx` will download the model in filesystem and then trace the model.\n",
+    "\n",
+    "after tracing the model (a .onnx file will be generated), `save_as_onnx` method zips `tokenizers.json` and torchScript (`.onnx`) file and saves in the file system. \n",
+    "\n",
+    "User can upload that model to opensearch to generate embedding.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "44d6b1d2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ONNX opset version set to: 16\n",
+      "Loading pipeline (model: sentence-transformers/all-MiniLM-L6-v2, tokenizer: sentence-transformers/all-MiniLM-L6-v2)\n",
+      "Creating folder /Volumes/workplace/upload_content/onnx\n",
+      "Using framework PyTorch: 1.13.1\n",
+      "Found input input_ids with shape: {0: 'batch', 1: 'sequence'}\n",
+      "Found input token_type_ids with shape: {0: 'batch', 1: 'sequence'}\n",
+      "Found input attention_mask with shape: {0: 'batch', 1: 'sequence'}\n",
+      "Found output output_0 with shape: {0: 'batch', 1: 'sequence'}\n",
+      "Found output output_1 with shape: {0: 'batch'}\n",
+      "Ensuring inputs are in correct order\n",
+      "position_ids is not present in the generated input list.\n",
+      "Generated inputs order: ['input_ids', 'attention_mask', 'token_type_ids']\n",
+      "zip file is saved to  /Volumes/workplace/upload_content/all-MiniLM-L6-v2.zip \n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "pre_trained_model = SentenceTransformerModel(folder_path = '/Volumes/workplace/upload_content/', overwrite = True)\n",
+    "pre_trained_model.save_as_onnx(model_id = \"sentence-transformers/all-MiniLM-L6-v2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "97ed5665",
+   "metadata": {},
+   "source": [
+    "## Step 4: Upload the saved Onnx model in Opensearch\n",
+    "\n",
+    "In the last step we saved a sentence transformer model in ONNX format. Now we will upload that model in opensearch cluster. To do that we can take help of `upload_model` method in `opensearch-py-ml` plugin.\n",
+    "\n",
+    "To upload model, we need the zip file we just saved in the last step and a model config file. Example of Model config file content can be:\n",
+    "\n",
+    "{\n",
+    "  \"name\": \"all-MiniLM-L6-v2\",\n",
+    "  \"version\": \"1.0.0\",\n",
+    "  \"description\": \"test model\",\n",
+    "  \"model_format\": \"ONNX\",\n",
+    "  \"model_config\": {\n",
+    "    \"model_type\": \"bert\",\n",
+    "    \"embedding_dimension\": 384,\n",
+    "    \"framework_type\": \"sentence_transformers\"\n",
```

Comment:
For ONNX model, user need to specify `pooling_mode` , `normalize_result`

Reply:
will add.